src/NuGet.Core/NuGet.Packaging.Extraction/NuGet.Packaging.Extraction.csproj

```diff
@@ -1,21 +1,69 @@
 ﻿<Project>
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), 'README.md'))\build\common.props" />
   <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
+  <Import Project="ilmerge.props" />
 
   <PropertyGroup>
     <Description>NuGet's understanding of packages. Reading nuspec, nupkgs and package signing.</Description>
     <TargetFrameworks>net45</TargetFrameworks>
-    <TargetFramework />
-    <NoWarn>$(NoWarn);CS1591;CS1574;CS1573;CS1572</NoWarn>
+    <TargetFramework></TargetFramework>
+    <NoWarn>$(NoWarn);CS1591;CS1574;CS1573;CS1572;NU5128</NoWarn>
     <PackProject>true</PackProject>
+    <SuppressDependenciesWhenPacking>true</SuppressDependenciesWhenPacking>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+    <TargetsForTfmSpecificContentInPackage>$(TargetsForTfmSpecificContentInPackage);ILMergeNPE;CreateNupkgFromILMerge</TargetsForTfmSpecificContentInPackage>
```

Comment:
It wouldn't change the outcome, but wouldn't it be more correct to make `CreateNupkgFromILMerge` DependsOnTargets="ILMergeNPE"?

Although, without using the MSBuild Inputs and Outputs attributes to implement incremental builds, it will probably mean that packing will ilmerge again, and therefore lose the assembly signing.

Reply:
actually, if we added another targetframework, we would want each DLL to be ILMerged...so I think it works well in line 14.